src/App.css

```diff
@@ -161,6 +190,46 @@
   margin: 0;
 }
 
+.AddBook h2 {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.addbookform {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.add-btn {
+  padding: 15px 4%;
+  background-color: #0290ff;
+  outline: none;
+  border: none;
+  border-radius: 3px;
+  color: #fff;
+}
+
+.inputfields {
+  width: 35%;
+  height: 50px;
+  padding: 0 20px;
+  margin-right: 10px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  letter-spacing: -0.15px;
+}
```

Comment:
Your final design should match the zeplin's design for both styling and the elements in order to prove your compentency to recruiters and other clients. You have really done well so far. However, there are a few issues you need to check as shown below. 

- [ ] Kindly check the horizontal line between the last bookcard and the form, it should always be in line and same width with them but in your design it sometimes get bigger or smaller as shown below. Kindly ensure that the bookcards, the horizontal line and the form all have the same width as indicated in the design. :pray: 
- [ ] There is a vertical line between the `comments` `Remove` and `Edit` buttons. It should also maintain the margin and paddings in the original design but yours is different. 


### Your design 

![Screenshot from 2023-06-13 18-30-31](https://github.com/misalazeem/bookstore-react/assets/65233860/83ad38b8-2552-47fe-aaa3-608260b91323)

![Screenshot from 2023-06-13 18-48-27](https://github.com/misalazeem/bookstore-react/assets/65233860/809a0005-6f47-49c3-b994-b3c6737796c8)

![Screenshot from 2023-06-13 18-48-43](https://github.com/misalazeem/bookstore-react/assets/65233860/87a2dde3-d369-4149-b816-4ac75d4efed4)



### Actual design 


![Screenshot from 2023-06-13 18-30-41](https://github.com/misalazeem/bookstore-react/assets/65233860/e098463f-c506-406f-8c1e-6ed9d05e0e4f)